src/pages/Cart/Cart.jsx

```diff
@@ -1,13 +1,91 @@
 import React from 'react'
-import { NavBar } from '../HomePage/HomePageComponents/NavBar/NavBar'
+import { NavBar } from '../HomePage/HomePageComponents/NavBar/NavBar';
+import {Footer} from '../HomePage/HomePageComponents/Footer/Footer';
+import "../../Colours/colours.css";
+import "../Cart/Cart.css"
+import { cartItems } from '../../backend/db/cart-items';
 
 function Cart() {
 return (
+
 <div>
-    <NavBar/>
-    <h1>Blank Cart page for Route checking</h1>
+
+    <NavBar />
+    
+    <div className="cart-container">
+
+        <h1 className="cart-header">My Cart</h1>
+        <div className="item-price-container-row">
+            {
+            cartItems.map(item=>{
+            return <div className="cart-card">
```

Comment:
Here you can create a new component like cardProduct and map over that.It will make your code look clean.

Reply:
Sure, I will 👍️